app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -75,6 +77,16 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
                     AboutFragment.newInstance()
                 }
             }
+            resources.getString(R.string.title_share_the_app) ->{
+                val shareIntent = Intent(Intent.ACTION_SEND)
+                shareIntent.type = "text/plain"
+                val shareBody = "Check This Out"
```

Comment:
`Check this out` instead of `Check This Out`. 

Could you also move this hardcoded string to all the strings.xml?